### PR TITLE
Clarify argument & variable names related to ADT vs MRN indexing in tensorizer

### PR DESCRIPTION
**Pull request by erikr**
## Changelog
- [x] Does not alter functionality
- [x] Revises arguments, variable names, and comments to be clearer.

## Checklist
- [x] The following are complete:
    - Added or revised tests if appropriate.
    - If new packages are installed, `requirements.txt` is updated.
    - If documentation should be updated, a new issue was made in `ml4c3-wiki`.
    - The PR name describes its content and does not contain "Ref #".
    - The PR does not add PHI.

### CODE REVIEW

File: ml4c3/arguments.py [hunk 1]
```diff
@@ -259,28 +259,26 @@ def parse_args() -> argparse.Namespace:
         "location.",
     )
     icu_parser.add_argument(
-        "--adt_start_mrn",
+        "--mrn_start_index",
         type=int,
         default=0,
         help="MRN of first patient in ADT table to process, inclusive.",
```
Comment (RogerPallares) [inline ml4c3/arguments.py:265]: May it be more clarifying: Index of first MRN in ADT table to process, inclusive. ?

File: ml4c3/arguments.py [hunk 2]
```diff
@@ -259,28 +259,26 @@ def parse_args() -> argparse.Namespace:
...
     )
     icu_parser.add_argument(
-        "--adt_end_mrn",
+        "--mrn_end_index",
         type=int,
-        default=None,
         help="MRN of last patient in ADT table to process, exclusive.",
```
Comment (RogerPallares) [inline ml4c3/arguments.py:270]: Same as in the other argument.

File: ml4c3/arguments.py [hunk 3]
```diff
@@ -259,28 +259,26 @@ def parse_args() -> argparse.Namespace:
...
+        "--mrn_start_index",
         type=int,
         default=0,
         help="MRN of first patient in ADT table to process, inclusive.",
     )
     icu_parser.add_argument(
-        "--adt_end_mrn",
+        "--mrn_end_index",
         type=int,
-        default=None,
```
Comment (RogerPallares) [inline ml4c3/arguments.py:270]: Why are you deleting the defaults? Does not writing a default create a `None` default?

**Review by RogerPallares — CHANGES_REQUESTED** [review]
Great, thanks for helping on that! Just a couple of minor changes.

File: ml4c3/arguments.py (hunk 1 above)
Comment (erikr) [reply ml4c3/arguments.py:265]: Good idea will update

File: ml4c3/arguments.py (hunk 3 above)
Comment (erikr) [reply ml4c3/arguments.py:270]: Yes!

**Review by RogerPallares — APPROVED** [review]

**Pull request merged**